diff/diff.go

```diff
@@ -99,12 +83,23 @@ const (
 )
 
 func operation(row string) Operation {
-	if strings.HasPrefix(row, "+") {
+	if strings.HasPrefix(row, Add.String()) {
 		return Add
 	}
-	if strings.HasPrefix(row, "-") {
+	if strings.HasPrefix(row, Delete.String()) {
 		return Delete
 	}
 
 	return Equal
 }
+
+func (o Operation) String() string {
```

Comment:
I thought it was a bit weird that you aren't handling Equal here.  I think adding a third case statement even if it just returns "" would be a bit clearer even if it's duplicate code.  My first instinct looking at the code was that you forgot to handle Equal here.

Reply:
sounds good, i can do that